types/helpers.go

```diff
@@ -317,25 +317,6 @@ func (s *String) UnmarshalBinary(data []byte) error {
 	return err
 }
 
-type Bool bool
```

Comment:
(I've added this in AC-586, but it's not consistent to use anymore)

Reply:
:+1: